src/main/webapp/app/admin/organization-management/organization-management.service.ts

```diff
@@ -22,8 +22,8 @@ export class OrganizationManagementService {
      * Send GET request to retrieve the number of users and courses of
      * all organizations
      */
-    getNumberOfUsersAndCoursesOfOrganizations(): Observable<any> {
-        return this.http.get(this.resourceUrl + '/count-all');
+    getNumberOfUsersAndCoursesOfOrganizations(): Observable<[{ organizationId: number; numberOfUsers: number; numberOfCourses: number }]> {
```

Comment:
Can you define a proper Typescript class for this? We would like to avoid such inline objects, it's also code duplication.

The Typescript class could be similar to OrganizationCountDTO.
Have a look at other DTOs such as `ExamScoreDTO` in `exam-score-dtos.model.ts`

Reply:
DTO model has been added with 97e6bbb

Thanks :+1: